Routing/Loader/RestXmlCollectionLoader.php

```diff
@@ -70,6 +70,7 @@ protected function parseNode(RouteCollection $collection, \DOMElement $node, $pa
                 $namePrefix = (string) $node->getAttribute('name-prefix');
                 $parent = (string) $node->getAttribute('parent');
                 $type = (string) $node->getAttribute('type');
+                $host = isset($config['host']) ? $config['host'] : null;
```

Comment:
@stloyd I've assigned to a local variable use the same structure that is currently present